internal/cluster/cluster.go

```diff
@@ -631,7 +631,7 @@ func (m *Manager) AcceptRegistration(c *common.Cluster) (err error) {
 	allowedStatuses := []string{models.ClusterStatusInsufficient, models.ClusterStatusReady, models.ClusterStatusPendingForInput, models.ClusterStatusAddingHosts}
 	if !funk.ContainsString(allowedStatuses, clusterStatus) {
 		if clusterStatus == models.ClusterStatusInstalled {
-			err = errors.Errorf("Cannot add host to a cluster that is already installed, please use the day2 cluster option")
+			err = errors.Errorf("Cannot add hosts to an existing cluster using the original Discovery ISO. Try to add new hosts by using the Discovery ISO that can be found in console.redhat.com under your cluster “Add hosts” tab.")
```

Comment:
```suggestion
			err = errors.Errorf("Cannot add hosts to an existing cluster using the original Discovery ISO. Try adding new hosts by using the Discovery ISO found in console.redhat.com under your cluster “Add hosts” tab.")
```

Reply:
also, we have a knowledge of the cluster id here: maybe we can display the tab URL for this cluster directly?

Reply:
OCM is using a different ID.